sdks/python/apache_beam/examples/inference/multi_language/last_word_prediction/src/main/java/org/MultiLangRunInference.java

```diff
@@ -0,0 +1,93 @@
+
+import java.io.IOException;
+import org.apache.beam.model.pipeline.v1.ExternalTransforms;
+import org.apache.beam.runners.core.construction.External;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.coders.RowCoder;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.Validation.Required;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.schemas.Schema.Field;
+import org.apache.beam.sdk.schemas.Schema.FieldType;
+import org.apache.beam.sdk.schemas.SchemaTranslation;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.util.ByteStringOutputStream;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.Row;
+import org.apache.beam.sdk.values.PDone;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MultiLangRunInference {
+    public interface MultiLangueageOptions extends PipelineOptions {
+
+        @Description("Path to an input file that contains labels and pixels to feed into the model")
+        @Required
+        String getInputFile();
+
+        void setInputFile(String value);
+
+        @Description("Path to an input file that contains labels and pixels to feed into the model")
+        @Required
+        String getOutputFile();
+
+        void setOutputFile(String value);
+    }
+
+    private static byte[] toStringPayloadBytes(String model) {
+        Row configRow = Row.withSchema(Schema.of(Field.of("model", FieldType.STRING)))
+                .withFieldValue("model", model)
+                .build();
+
+        ByteStringOutputStream outputStream = new ByteStringOutputStream();
+
+        try {
+            RowCoder.of(configRow.getSchema()).encode(configRow, outputStream);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        ExternalTransforms.ExternalConfigurationPayload payload = ExternalTransforms.ExternalConfigurationPayload
+                .newBuilder()
+                .setSchema(SchemaTranslation.schemaToProto(configRow.getSchema(), false))
+                .setPayload(outputStream.toByteString())
+                .build();
+        return payload.toByteArray();
+    }
+
+    public static void main(String[] args) {
+
+        final String TEST_RUN_BERT_URN = "beam:transforms:xlang:test:run_bert";
+        MultiLangueageOptions options = PipelineOptionsFactory.fromArgs(args).withValidation()
+                .as(MultiLangueageOptions.class);
+
+        Pipeline pipeline = Pipeline.create(options);
+        PCollection<String> predictions = pipeline.apply("Read Input", TextIO.read().from(options.getInputFile()))
+            .apply("Run Inference" ,External.of(TEST_RUN_BERT_URN, toStringPayloadBytes("bert-base-uncased"), "localhost:12345"));
```

Comment:
`localhost:12345` what is this? Is this the expansion service port? If so, should this be configurable/passed in as an arg?